data_preparation/CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Adding or onboarding data to AgML Subnational Crop Yield Forecasting
+Thank you for your interest in contributing to the AgML Subnational Crop Yield Forecasting benchmark. You could contribute one of the following
+1. Crop statistics for a country or region.
+2. Alternative or additional predictor data source.
+3. Alternative crop mask source.
+4. Alternative crop calendar source.
+
+## Contributing crop statistics
+1. First check if crop statistics for the country or region of interest is already included.
+2. If not, create an issue and follow the [contributing guidelines](../README.md). Create a new directory under `data_preparation`.
+3. Create a data card (`README.md`) following [the template](DATA-CARD-TEMPLATE.md) in the new directory.
+4. As much as possible, look for ways to automatically export and prepare the dataset. This way the steps can be followed by other community members to prepare the data. Use existing APIs if available. If not, document the steps required to export the data. Add a notebook or script to explore the data and to prepare it in [the expected format](DATA-FORMAT.md). Add the notebook or script in the new directory.
```

Comment:
@krsnapaudel where should the documentation of the steps required to download the data go?

Reply:
This can be in the `data_preparation` script or notebook.